packages/api/src/record.ts

```diff
@@ -295,6 +350,91 @@ export class Record implements RecordModel {
     return dataObj;
   }
 
+  private async _processMessage(options: any){
+
+    const { store = true } = options;
```

Comment:
Please add some documentation that explains why another `processMessage()` private method was added.

Reply:
Done